sdks/java/extensions/protobuf/src/test/java/org/apache/beam/sdk/extensions/protobuf/ProtoByteUtilsTest.java

```diff
@@ -144,4 +172,23 @@ public void testRowToProtoSchemaFunction() {
     Assert.assertNotNull(
         ProtoByteUtils.getRowToProtoBytesFromSchema(PROTO_STRING_SCHEMA, "MyMessage").apply(row));
   }
+
+  @Test
+  public void testRowToProtoSchemaWithPackageFunction() {
+    Row row =
+        Row.withSchema(SCHEMA)
+            .withFieldValue("id", 1234)
+            .withFieldValue("name", "Doe")
+            .withFieldValue("active", false)
+            .withFieldValue("address.city", "seattle")
+            .withFieldValue("address.street", "fake street")
+            .withFieldValue("address.zip_code", "TO-1234")
+            .withFieldValue("address.state", "wa")
+            .build();
+
+    Assert.assertNotNull(
+        ProtoByteUtils.getRowToProtoBytesFromSchema(
+                PROTO_STRING_PACKAGE_SCHEMA, "com.test.proto.MyMessage")
+            .apply(row));
```

Comment:
I think having at least 1 test case for `getRowToProtoBytesFromSchema` that validates the output proto after invoking `.apply()` would be beneficial (whether it be this test or one of the existing ones)